api/photoTimeApi.js

```diff
@@ -124,9 +160,17 @@ var gPhotoTimeAPI = (function() {
 		return getRepos;
 	};
 
+	var activeConnection = null;
+
 	return {
 		connect: function(server) {
-			return new PhotoTimeConnection(server, createRequest);
+			activeConnection = new PhotoTimeConnection(server, createRequest);
+			return activeConnection;
+		},
+		getConnection: function() {
+			return activeConnection
 		}
 	};
-})();
\ No newline at end of file
+})();
+
+export { gPhotoTimeAPI }
```

Comment:
add new line at end of file

Reply:
this is a very silly request, not worth my time. But in future commits I will check the files I'm editing to humor you. 

; )